Makefile

```diff
@@ -4,6 +4,9 @@
 ## Auto-generated, do not modify!
 ####################################################################################
 ####################################################################################
+
+include quiet.mk
```

Comment:
Can you rebase onto the latest dev_tupgrade. I think this line and the other quiet.mk include change should disappear in that case.

Reply:
Will merge this branch into master, after the dev_tupgrade will be merged. So, I would not worry about this just for now. (There definitely going to be a rebase at some point)

Or will this break something, that I did not find?

Reply:
Won't break anything. Looks just a bit strange to have this 'random' change part of that commit.